### DISCUSSION TITLE
can i merge two or more CroppedPage into one?

### DISCUSSION THREAD

**Discussion by chopin1998** [jsvine/pdfplumber#902] [Q&A]
Please describe, in as much detail as possible, your proposal and how it would improve your experience with pdfplumber.
hi, if i croped two parts, can i merge them into one?

**Comment by samkit-jain** [comment]
Hi @chopin1998 Appreciate your interest in the library. At this moment, no, merging two cropped parts into one is not possible. Could you please also elaborate on your use-case? Maybe there's a solution to it.
